cluster/examples/kubernetes/ceph/object.yaml

```diff
@@ -86,8 +86,8 @@ spec:
     #name: zone-a
   # service endpoint healthcheck
   healthCheck:
-    endpoint:
-      enabled: true
+    bucket:
+      disabled: false
```

Comment:
One last thing that came to my mind, is the Documentation also correct now?

Reply:
Yes!